internal/plugin/provider.go

```diff
@@ -118,11 +118,7 @@ func (p *AivenProvider) Resources(context.Context) []func() resource.Resource {
 
 	// Add to a list of resources that are currently in beta.
 	if util.IsBeta() {
-		betaResources := []func() resource.Resource{
-			organization.NewOrganizationApplicationUser,
```

Comment:
Could you please also initialize them above? The test is currently failing because those resources weren't initialized.

```
=== RUN   TestAccOrganizationApplicationUserResourceDataSource
    organization_application_user_test.go:31: Step 1/5 error: Error running pre-apply plan: exit status 1
        
        Error: Invalid resource type
        
          on terraform_plugin_test.tf line 16, in resource "aiven_organization_application_user" "foo":
          16: resource "aiven_organization_application_user" "foo" {
        
        The provider hashicorp/aiven does not support resource type
        "aiven_organization_application_user".
--- FAIL: TestAccOrganizationApplicationUserResourceDataSource (5.89s)
```

Reply:
That's why I want to refactor this behaviour.